Doc/library/pickle.rst

```diff
@@ -428,6 +435,34 @@ The :mod:`pickle` module exports two classes, :class:`Pickler` and
       :ref:`pickle-restrict` for details.
 
 
+.. class:: PickleBuffer(buffer)
+
+   A wrapper for a potentially out-of-band buffer.  *buffer* must be a
+   :ref:`buffer-providing <bufferobjects>` object, such as a
+   :term:`bytes-like object` or a N-dimensional array.
+
+   :class:`PickleBuffer` is itself a buffer provider, therefore it is
+   possible to pass it to other APIs expecting a buffer-providing object,
+   such as :class:`memoryview`.
```

Comment:
Add maybe a reference to the Buffer Protocol:
https://docs.python.org/dev/c-api/buffer.html#buffer-protocol

See "bytes-like object" glossary entry for an example of such link:
https://docs.python.org/dev/glossary.html

Reply:
There is a reference in the sentence just before.